sidebars-main.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
         'get-started/implementation-checklist',
       ],
     },
+    'job-writing-guide',
```

Comment:
@josephjclark I'm on board with this being more elevated in the sidebar, so long as it links to the more [detailed job-specific docs](https://docs.openfn.org/documentation/build/steps/jobs). Can you please consider how to link the 2 sections?

Reply:
This is really nice I think - makes the docs more discoverable and cohesive 
![image](https://github.com/OpenFn/docs/assets/7052509/af2eb430-f2e2-4638-a2b2-1a0387750c38)